documentation/decisions/adr/0010-front-end-framework.md

```diff
@@ -0,0 +1,96 @@
+# Front-end Framework
+
+- **Status:** Draft <!-- REQUIRED -->
+- **Last Modified:** 2023-07-13 <!-- REQUIRED -->
+- **Related Issue:** [#97](https://github.com/HHS/grants-api/issues/97) <!-- RECOMMENDED -->
+- **Deciders:** Lucas Brown, Aaron Couch, Billy Daly, Sammy Steiner, Daphne Gold <!-- REQUIRED -->
+
+## Context and Problem Statement
+
+The project will need a web framework to build and manage the frontend. The goal of this ADR is to evaluate and select a frontend web framework.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+### Must Haves
+- Active Maintenance: The web framework is actively maintained with patches and minor releases delivered on a regular basis
+- Community of Users: The web framework has an active community of open source users and the framework is commonly used for frontend development
+- Usability: The framework is relatively easy to learn for developers without prior experience and there are plenty of resources and training materials available
```

Comment:
```suggestion
- Usability: The framework is relatively easy to learn for developers without prior experience in the particular framework, and there are plenty of resources and training materials available
```

Or maybe explicitly call out JS experience: "The framework is relatively easy to learn for developers with JavaScript experience and there are plenty of resources and training materials available." Don't wanna sound like supporting zero dev experience is a must-have. 

</nit>